elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -220,24 +230,24 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
         Future.successful(acc)
       } else {
         bulkDelete(index, tpe, documentIds.map(Document(_, Map())))
-          .flatMap { items =>
-            scrollDelete(
-              index,
-              tpe,
-              id,
-              acc ++ items.map(item => Dsl.Index(item._id) -> DeleteResponse(item.success)),
-              (scrollId) => scroll(scrollId))
-          }
+            .flatMap { items =>
+              scrollDelete(
+                index,
+                tpe,
+                id,
+                acc ++ items.map(item => Dsl.Index(item._id) -> DeleteResponse("deleted")),
```

Comment:
seems like we used to report whether or not the delete was successful? Is that not information we have anymore?

Reply:
This is the response type : 
```
{
    "_shards" : {
        "total" : 2,
        "failed" : 0,
        "successful" : 2
    },
    "_index" : "twitter",
    "_type" : "tweet",
    "_id" : "1",
    "_version" : 2,
    "_primary_term": 1,
    "_seq_no": 5,
    "result": "deleted"
}
```